home/urls.py

```diff
@@ -1,10 +1,8 @@
-from django.contrib import admin
-from django.urls import path, include
-from django.views.generic.base import RedirectView
-from django.conf import settings
-from django.conf.urls.static import static
+from django.urls import path
 from .views import HomeView
+from . import views
 
 urlpatterns = [
     path('', HomeView.as_view(), name='index'),
+    path('category/',views.CategoryView.as_view(), name='cate'),
```

Comment:
Url name nên đặt tên rõ ràng e nhé, ko viết tắt để tránh gây hiểu lầm
```suggestion
    path('category/',views.CategoryView.as_view(), name='category'),
```